CONTRIBUTING.md

```diff
@@ -33,29 +33,35 @@ cargo clippy --workspace --tests -- --deny warnings
 
 Execute `cargo fmt --all` to fix the formatting.
 
-## Generating Docs
-
-If you make changes to [Roc's Standard Library](https://www.roc-lang.org/builtins/Str), you can add comments to the code following [the CommonMark Spec](https://spec.commonmark.org/current/) to further explain your intentions. You can view these changes locally with:
-
-```sh
-cargo run docs crates/compiler/builtins/roc/main.roc
-```
-
-This command will generate the documentation in the [`generated-docs`](generated-docs) directory.
-
 ## Contribution Tips
 
 - If you've never made a pull request on github before, [this](https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/) will be a good place to start.
-- Create an issue if the purpose of a struct/field/type/function/... is not immediately clear from its name or nearby comments.
```

Comment:
I added this but nobody does it, so there's no point in keeping it.

Reply:
Might be worth linking to https://www.roc-lang.org/community#ideas somewhere??

Reply:
I even started refactoring and documenting the `Defs` SOA because of that. Not sure if an issue is necessary tho.

Reply:
Interesting, to make sure I understand correctly @kdziamura; did you do the refactoring and documenting because you read the "Create an issue if the purpose of a struct/field/type/function/... is not immediately clear from its name or nearby comments."?

> Not sure if an issue is necessary tho.

Should we recommend asking on zulip instead?

Reply:
> did you do the refactoring and documenting because you read the "Create an issue if the purpose of a struct/field/type/function/... is not immediately clear from its name or nearby comments."?

Nope. I mean, I started messing with it because I come across the struct repeatedly and even though I already understand it completely, I want to make life easier for myself and other newcomers.

I read the md file a long time ago, and I don't remember about the suggestion to create an issue. I think for the matter of quick light messaging, zulip works the best. However, it would be nice if such conversations end up with better documentation in the repo, and issues are the best for that (to track the problem).

Reply:
People do look at the good first issues, so perhaps we should create one for adding documentation? Like find a file/struct/function where it's not clear what it does, try to figure it out, ask us on zulip and make a PR that adds documentation for that thing.